code/_defines/items.dm

```diff
@@ -1,8 +1,15 @@
 #define ITEM_SLOT_RING	"ring"
 #define ITEM_SLOT_HANDS	"hands"
+#define ITEM_SLOT_HEAD	"head"
+#define ITEM_SLOT_CHEST	"chest"
+#define ITEM_SLOT_BELT	"belt"
+#define ITEM_SLOT_LEGS	"legs"
+#define ITEM_SLOT_CLOTHING	"clothing"
```

Comment:
need to think a bit about inventory stuff still, but since there is no proper inventory yet it is fine.